src/main/java/com/panasi/interview_questions/repository/dto/CategoryDto.java

```diff
@@ -1,12 +1,15 @@
 package com.panasi.interview_questions.repository.dto;
 
+import java.util.List;
+
 import lombok.Data;
 
 @Data
 public class CategoryDto {
 	
 	private Integer id;
-	private Integer parentId;
 	private String name;
+	private Integer parentId;
+	private List<QuestionDto> questions;
```

Comment:
I am not sure that we need here questions objects, it can be a very big object, when we only will look category tree
better make additional request to get all questions by category

Reply:
Removed list of questions from category DTO